api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/activation/Activations.kt

```diff
@@ -419,3 +432,37 @@ public class HardShrinkActivation(public val lower: Float = -0.5f, public val up
         }
     }
 }
+
+/**
+ * @see [Activations.Gelu]
+ */
+public class GeluActivation(public val approximate: Boolean = false) : Activation {
+    override fun apply(tf: Ops, features: Operand<Float>): Operand<Float> {
+        if (approximate) {
+            val coeff = tf.constant(0.044715f)
+            return tf.math.mul(
+                tf.constant(0.5f), tf.math.mul(
+                    features, tf.math.add(
+                        tf.constant(1.0f), tf.math.tanh(
+                            tf.math.mul(
+                                tf.constant(0.7978845608028654f),
```

Comment:
Looks like optimization to avoid the pi transformation?

Reply:
The value is of `sqrt(2/pi)` to avoid a constant division i took this value. This is similar to tf.nn.gelu implementation. I could add in sqrt if you want?

Reply:
maybe better to add it in the comments, for the future code readers